sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -58,6 +65,35 @@ def run_inference(
     for example in batch:
       yield model.predict(example)
 
+  def update_model_path(self, model_path: Optional[str] = None):
+    pass
+
+
+class FakeModelHandlerReturnsPredictionResult(
+    base.ModelHandler[int, base.PredictionResult, FakeModel]):
+  def __init__(self, clock=None, model_id='fake_model_id_default'):
+    self.model_id = model_id
+    self._fake_clock = clock
+
+  def load_model(self):
+    if self._fake_clock:
+      self._fake_clock.current_time_ns += 500_000_000  # 500ms
+    return FakeModel()
```

Comment:
Right now, this test doesn't actually show that this load_model method is getting invoked since this is basically a no-op after the first go-round. Could we rewrite this test so that FakeModel (or different fake model class) takes in the updated model id here and uses that to modify what it returns for its prediction?

Reply:
Added 2 models, addition and sub but in a different IT test.